h/httpfileservice.h

```diff
@@ -15,6 +15,17 @@
 
 #include "httpserver.h"
 
+#define HTTP_FILE_SERVICE_SUCCESS                0 
```

Comment:
The constants are statuses or return codes. Can you make it clear by adding `RC` or `STATUS` into the names.

Reply:
Added RC prefix to the macros.